projects/angular/src/data/datagrid/_datagrid.clarity.scss

```diff
@@ -1470,9 +1470,19 @@
       .column-toggle--action {
         // I'm overriding .btn/.btn-link here but am not confident this is the correct way to do it.
         min-width: variables.$clr_baselineRem_0_75;
-        padding-left: variables.$clr_baselineRem_6px;
-        padding-right: variables.$clr_baselineRem_6px;
         text-transform: capitalize;
+        @include mixins.css-var(
+          gap,
+          clr-datagrid-column-toggle-gap,
+          variables.$clr_baselineRem_4px,
+          variables.$clr-use-custom-properties
+        );
+        @include mixins.css-var(
+          padding,
+          clr-datagrid-column-toggle-padding,
+          0 variables.$clr_baselineRem_6px,
+          variables.$clr-use-custom-properties
+        );
```

Comment:
Do we need both of these? It seems we didn't have the gap before and keeping either one of them should be good.

Reply:
We need both for this iteration.

Tokens require `gap`. Original NG require `padding`.
Once we migrate the tokens only the gap will remain.